sops/alerts/apicast_worker_restart.adoc

```diff
@@ -0,0 +1,22 @@
+:toc:
+:toc-placement!:
+
+= ThreescaleApicastWorkerRestart
+
+toc::[]
+
+== Description
+
+* This alert will trigger if one of the nginx-worker restarted for any reason.
+
+== Troubleshooting
+
+* Check why the worker died in the Apicast logs; the usual reasons are:
+  - Pod hits the memory limits, and Cgroups kill one of the worker processes.
+  - Node is out of memory, so try to kill containers to schedule to another node.
+
+== Verification
+* If the kill happens because pod limits, check why the limits were reached,
+  and if it's needed, increase the memory.
+* Suppose node killed the container, nothing to do. This behaviour will be
+  changed in the next releases.
```

Comment:
```suggestion
* Suppose node killed the container, nothing to do. This behaviour will be
  changed in the next releases.
```

This also need to go the troubleshooting section and if you can add a bit more context so why a node may kill the container, it would be great.